colp_build/sql/name.sql

```diff
@@ -0,0 +1,110 @@
+		                                                                            '\s(GROUP 1)\s'
+		                                                                            ),
+		                                                                        '\sPK\s|\sPK-',
+		                                                                        ' PARK '
+		                                                                        ),
+		                                                                    'STA\s',
+		                                                                    'STATION '
+		                                                                    ),
+		                                                                'LIBR\s',
+		                                                                'LIBRARY '
+		                                                                ),
+		                                                            'INDUST\sPARK|IND\.\sPARK',
+		                                                            'INSUSTRIAL PARK'
+		                                                            ),
+		                                                        'PCT\s|PCT\.\s',
+		                                                        'PRECINCT '
+		                                                        ),
+		                                                    'N\.Y\.C\.P\.D\.|N\.Y\.P\.D\.|NYCPD',
+		                                                    'NYPD'
+		                                                    ),
+		                                                'F\.H\.A\.\.',
+		                                                'FHA '
+		                                                ),
+		                                            'GDN\s|GARD\s',
+		                                            'GARDEN '
+		                                            ),
+	                                            'AVE\)|AVE\.\)',
+	                                            'AVENUE)'
+	                                            ),
+                                            '\(AVE|\(AVE\.',
+                                            '(AVENUE'
+                                            ),
+                                        '\sAVE\s|\sAVE\.\s|\sAVE-\s|\sAVE\.\s|\sAV\s',
+                                        ' AVENUE '
+                                        ),
+                                    'HUNTS%POINT%AVENUE%REHAB',
+                                    'HUNTS POINT AVENUE REHAB'
+                                    ),
+                                'AUTH\s',
+                                'AUTHORITY '
+                                ),
+                            'BKLN|BK|BKLYN',
+                            'BROOKLYN'
+                            ),
+                        '\sBX\s|\sBRNX\s|/BX\s|\sBX\.\s',
+                        ' BRONX '
```

Comment:
PBBX PARKING LOT is becoming PBBRONX PARKING LOT and I don't think that's an accurate transformation